source/jsonnet/northern-ireland/individual/blocks/identity-and-health/arrive_in_country.jsonnet

```diff
@@ -22,6 +22,8 @@ local question(title) = {
       validation: {
         messages: {
           MANDATORY_DATE: 'Enter a year of arrival',
```

Comment:
This could be removed, as the question is not mandatory.

Reply:
Thanks!